test/groovy/PiperTestBase.groovy

```diff
@@ -0,0 +1,24 @@
+import com.lesfurets.jenkins.unit.BasePipelineTest
+
+import static ProjectSource.projectSource
+import static com.lesfurets.jenkins.unit.global.lib.LibraryConfiguration.library
+
+public class PiperTestBase extends BasePipelineTest {
+
+    protected final void _setUp() {
```

Comment:
Would be more convenient to have simply `setUp()`. But that does not work. In case we do so and we call `super.setup()` from a inheriting class, super is not properly resolved. Instead we get `this`, but not `super`.